alarms.json.default

```diff
@@ -20,7 +20,7 @@
 			"to_number":"YOUR_TO_NUM"
 		},
 		{
-			"active": "True",
+			"active": "False",
```

Comment:
Not sure but this was set to true in the default config file, while all others are false. Pretty sure you want them all to be false.